app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
+  has_many :customer_items
+  has_many :customers, through: :customer_items
+
+  def count_customers
+    customers.count
```

Comment:
Nice work encapsulating this logic at the model level. Even though it's a simple calculation, per MVC conventions, it's data calculation/logic so it should live here!

Reply:
Make sure to add `.distinct` next time! Repeat customers would be counted twice